src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -104,115 +116,31 @@ private String datasetName() {
-        assertThat(
-            String.format("Table: %s.file, exists", dataset.getName()),
-            fileTableExists,
-            equalTo(shouldExist));
-        String fileSoftDeleteTableName =  bigQueryPdao.prefixSoftDeleteTableName("file");
-        boolean fileSoftDeleteTableExists =  bigQueryPdao.tableExists(dataset, fileSoftDeleteTableName);
-        assertThat(
-            String.format("Table: %s.%s, exists", dataset.getName(), fileSoftDeleteTableName),
-            fileSoftDeleteTableExists,
-            equalTo(shouldExist));
-    }
-
-    private List<String> getSoftDeletedRowIds(Dataset dataset,
-                                           String tableName,
-                                           String projectId,
-                                           BigQueryProject bigQueryProject) {
-
-        StringBuilder builder = new StringBuilder();
-        builder.append("SELECT ")
-            .append(PDAO_ROW_ID_COLUMN)
-            .append(" FROM `")
-            .append(projectId)
-            .append(".")
-            .append(bigQueryPdao.prefixName(dataset.getName()))
-            .append(".")
-            .append(bigQueryPdao.prefixSoftDeleteTableName(tableName))
-            .append("`");
-
-        String sql = builder.toString();
-        TableResult result = bigQueryProject.query(sql);
-
-        List<String> rowIds = new ArrayList<>();
-        for (FieldValueList row : result.iterateAll()) {
-            String rowId = row.get(0).getStringValue();
-            rowIds.add(rowId);
-        }
-
-        return rowIds;
-    }
-
-    private List<String> getRowIds(Dataset dataset,
-                                   String tableName,
-                                   String projectId,
-                                   BigQueryProject bigQueryProject) {
-        StringBuilder builder = new StringBuilder();
-        builder.append("SELECT ")
-            .append(PDAO_ROW_ID_COLUMN)
-            .append(" FROM `")
-            .append(projectId)
-            .append(".")
-            .append(bigQueryPdao.prefixName(dataset.getName()))
-            .append(".")
-            .append(tableName)
-            .append("` EXCEPT DISTINCT (SELECT ")
-            .append(PDAO_ROW_ID_COLUMN)
-            .append(" FROM `")
-            .append(projectId)
-            .append(".")
-            .append(bigQueryPdao.prefixName(dataset.getName()))
-            .append(".")
-            .append(bigQueryPdao.prefixSoftDeleteTableName(tableName))
-            .append("`)");
-
-        String sql = builder.toString();
-        TableResult result = bigQueryProject.query(sql);
-
-        List<String> rowIds = new ArrayList<>();
-        for (FieldValueList row : result.iterateAll()) {
-            String rowId = row.get(0).getStringValue();
-            rowIds.add(rowId);
-        }
-
-        return rowIds;
+        Arrays.asList("participant", "sample", "file").forEach(name -> {
```

Comment:
If I understand correctly this is exactly the same test as before but avoiding duplication. Awesome!